app/views/jobs/index.html.erb

```diff
@@ -13,10 +13,8 @@
   </thead>
   <tbody>
     <% @groups.count.each do |k,v|%>
-      <% joblist = Job.where("name= ?", k).order("created_at DESC") %>
-      <% job_condition = joblist.first["created_at"] + joblist.first["frequency"]*3600 >= Time.now  && joblist.first["status"] == "success" %>
-          <tr class="<%=job_condition ? 'all_success' : 'not_success' %> ">
-            <td><%= job_condition ? "&#10003;".html_safe : "&#10008;".html_safe  %></td>
+          <tr class="<%= is_status_good(k) ? 'all_success' : 'not_success' %> ">
```

Comment:
If "is_status_good" sends an e-mail (even if delayed 8 hours) won't there be an e-mail every time someone looks at the index page when there's a bad status?  What we want is something that isn't dependent on user interaction with the web site.   Maybe create a rake task that checks whether any jobs are in in a failed condition and then sends and e-mail.  We'll then have a cron job that calls that rake task periodically.